### PR TITLE
feat: ability to update customer item deadlines

### PR DESCRIPTION
**Pull request by AdrianAndersen**

### CODE REVIEW

**Comment by AdrianAndersen** [comment]
[Mulighet for å endre frist for customerItems](https://trello.com/c/rSYyTIQs/715-mulighet-for-%C3%A5-endre-frist-for-customeritems)

File: src/app/customer-item/customer-item-detail/customer-item-detail.component.ts [hunk 1]
```diff
@@ -53,22 +62,52 @@ export class CustomerItemDetailComponent implements OnInit {
...
+			this.toPrettyDate(this.customerItem.deadline) !== this.deadlineInput
+		);
+	}
+
+	public async handleUpdateDeadline() {
+		try {
+			this.customerItem = await this._customerItemService.update(
+				this.customerItem.id,
+				{
+					deadline: moment(this.deadlineInput, "DD.MM.YYYY"),
```
Comment (LarsSelbekk) [inline src/app/customer-item/customer-item-detail/customer-item-detail.component.ts:101]: This is logged when you try to save an invalid deadline; I guess it's fine, though I would prefer not trying to update if moment returns null.
```
silly -> {"deadline":null}
error failed to save document: ValidationError: deadline: Path `deadline` is required.
verbose ! (701): failed to save the document
verbose ! (701) {"deadline":null}
verbose ! (701) {"errors":{"deadline":{"name":"ValidatorError","message":"Path `deadline` is required.","properties":{"message":"Path `deadline` is required.","type":"required","path":"deadline","value":null},"kind":"required","path":"deadline","value":null}},"_message":"customeritems validation failed","name":"ValidationError","message":"customeritems validation failed: deadline: Path `deadline` is required."}
```

**Review by LarsSelbekk — APPROVED** [review]
Pretty good, single nitpicky comment. MAW

File: src/app/customer-item/customer-item-detail/customer-item-detail.component.ts (hunk 1 above)
Comment (AdrianAndersen) [reply src/app/customer-item/customer-item-detail/customer-item-detail.component.ts:101]: oh well, that is kinda how things are handled in bl-admin. In bl-next admin, we can improve

**Pull request merged**